mindsdb/integrations/handlers/pytorch_tabular_handler/pytorch_tabular_handler.py

```diff
@@ -23,21 +25,26 @@ def create_validation(self,args: Optional[dict] = None) -> None:
         task_supported = ['regression','classification','backbone']
         initialization_supported = ['kaiming','xavier','random']
         args = args["using"]
-        if "task" not in args:
+        #pattern for layers regex
+        pattern = r'^\d+-\d+-\d+$'
+        if "task" in args:
             if args["task"] not in task_supported:
                 raise Exception(f"Please specify task parameter supported : {task_supported}")
         if "initialization" not in args or args["initialization"] not in initialization_supported:
             raise Exception(f"Initialization scheme choices are : {initialization_supported}")
-        #if not target:
-            #raise Exception("Please provide the target column")
+        if "layers" in args:
+            if not re.match(pattern,args["layers"]):
+                raise Exception(f"Please specify layers in format : '128-64-32'")
```

Comment:
Out of ignorance (no experience with pytorch tabular yet), why is a formatted string preferred to a list of integers?

Our SQL parser does support something like this:

```sql
CREATE MODEL
...
USING layers = ['128', '64', '32']  -- integers also work: [128, 64, 32] 
```